pkg/rancher-desktop/pages/snapshots/create.vue

```diff
@@ -0,0 +1,153 @@
+<script lang="ts">
+
+import { Banner, LabeledInput, TextAreaAutoGrow } from '@rancher/components';
+import Vue from 'vue';
+
+const defaultName = () => {
+  const dateString = new Date()
+    .toISOString()
+    .slice(0, 19)
+    .replace('T', '_')
+    .replace(/-/g, '_')
+    .replace(/:/g, '_');
+
+  return `snap_${ dateString }`;
+};
+
+interface Data {
+  name: string,
+  notes: string,
+  creating: boolean,
+}
+
+export default Vue.extend<Data, any, any, any>({
```

Comment:
Similar to others, we should be able to allow Vue to infer types and remove the usage of `any` if we provide type annotations for the data prop and each of the computed props in this component.